css/style.css

```diff
@@ -0,0 +1,40 @@
+/* #image{
+    margin: auto;
+    max-width:fit-content;
+    max-height: fit-content;
+
+}
+body{
+    text-align: center;
+}
+li{
+    text-align: left;
+    list-style-position: inside;
+} */
```

Comment:
- [ ] Don't leave commented lines of code if you don't need them, delete them Immediately.